src/sentry/api/endpoints/organization_release_previous_commits.py

```diff
@@ -44,6 +47,12 @@ def get(self, request, organization, version):
             .extra(order_by=["-date"])[:1]
         )
 
+        analytics.record(
+            "releases.set-commits.local",
+            organization_id=organization.id,
```

Comment:
Will it be a problem to query given that this one doesn't have a `user_id` and `project_id`?

Reply:
releases are at the org level, right?

Reply:
@manuzope updated to include those fields.
@szeitlin not sure what you mean? A release have 1 org but can have many associated projects.

Reply:
Yes, but AFAIK the way it is right now, by default releases are associated with orgs, not projects. So we don't have an easy way to tell whether a project actually changed in a given release or not, we just kind of assume that all projects in an org are associated with that release.

Reply:
@szeitlin ah! we can specify when we create a new release which projects to associate. It won't be all the projects in the org unless you want that.